src/state/commons/ScenarioConstants.js

```diff
@@ -22,4 +22,5 @@ export const SCENARIO_ACTIONS_KEY = {
   SET_SCENARIO_NAME: 'SET_SCENARIO_NAME', // Use to update reducer by setting scenario name in scenarios list
   START_SCENARIO_STATUS_POLLING: 'START_SCENARIO_STATUS_POLLING',
   STOP_SCENARIO_STATUS_POLLING: 'STOP_SCENARIO_STATUS_POLLING',
+  STOP_ALL_SCENARIO_STATUS_POLLING: 'STOP_ALL_SCENARIO_STATUS_POLLING',
```

Comment:
just a question: is it an action stopping polling of all scenarios ? If yes, typo: STOP_ALL_SCENARIO**S**_STATUS_POLLING :slightly_smiling_face:

Reply:
yes modified to STOP_ALL_SCENARIO_STATUS_POLLING